src/duckdb/json.rs

```diff
@@ -51,7 +51,7 @@ impl OptionValidator for JsonOption {
     }
 }
 
-pub fn create_view(
+pub fn create_duckdb_relation(
```

Comment:
Why are you renaming these functions? This again is making the diff much larger and harder to review. You're not even changing the code of the function, so why does it need a new name?